crates/cdk/Cargo.toml

```diff
@@ -58,6 +58,7 @@ uuid = { version = "1", features = ["v4", "serde"] }
 # -Z minimal-versions
 sync_wrapper = "0.1.2"
 bech32 = "0.9.1"
+chrono = "0.4.39"
```

Comment:
Imports under the -Z minimal-versions are for the case where a dep us our does not properly define their minimum deps I don't think that is the case here. 

Do we need the dep in cdk I don't see where it is used?